extension/test/integration/lib/bigcommerce/CustomerRepository-test.js

```diff
@@ -0,0 +1,66 @@
+const assert = require('assert')
+const chai = require('chai')
+chai.use(require('chai-as-promised')).should()
+const BigCommerceCustomerRepository = require('../../../../lib/bigcommerce/CustomerRepository')
+const BigCommerceRequestClientError = require('../../../../lib/bigcommerce/customer/request/ClientError')
+const { clientId, accessToken, storeHash } = require('../../../../../.integration-credentials')
```

Comment:
integration-credentials should be in the extension directory (like in the bc-cart extension)
```suggestion
const { clientId, accessToken, storeHash } = require('../../../../.integration-credentials')
```